---
deprojection_pipeline/ObjectPositions result